### ISSUE TITLE
Log Spam & lack of function on Slackware-Current

### ISSUE THREAD

**Issue by BusinessKid** [Rafficer/linux-cli-community#369]
I had the CLI version 2.2.11 working fine on Slackware64-15.0, until a clean install became necessary. I updated to current, and installed the Slackbuild.Now 2.2.12 on Current (2024-03-22) it generates copious log spam (attached is o/p of 'protonvpn c --cc ie' [protonvpn.txt](https://github.com/Rafficer/linux-cli-community/files/14802571/protonvpn.txt)

Version 2.2.12 actually connects, to judge by ifconfig, /etc/resolv.conf & the routing table. I "Upgraded" back to 2.2.11, which  spouts similar nonsense and doesn't work. Slackware-Current comes with an impressive array of python modules. The version is python3-3.9.19. I installed the pythondialog & docopt modules with pip3, but I'm presuming there's more required. One error I see repeated is that it apparently can't parse '\u2014' with some ascii module . I am not prepared to ignore that level of log spam. Is there a list of (python3) requirements?

**Comment by BusinessKid** [comment]
I found the requirements.txt in the archive, and all the requirements were already satisfied.

**Comment by BusinessKid** [comment]
Right. Haven't heard from you, but with the help of the LQ community, I've a solution, or workaround at least.
I  reinstalled your requirements with pip3, reinstalled python-importlib_metadata which is supplied as a slackware package in their ~Current distro, and tried it. I get 70k of errors [https://pastebin.com/W7Aj66Za](url) but the connect and disconnect work, and I can get by with those two commands. I checked function with whatismyip.com, and it's changing. So function is not impared. I edited my init script to read `protonvpn connect --cc IE 2>/dev/null`  and connection is readable.

I suspect it's a python bug, as python3.9x worked ok, so I'm closing this The protonvpn lot are Swiss, or based there at least and Europe is  Unicode country. So it's unlikely  they would tolerate having Unicode so badly wrong.
